python/ray/tune/search/optuna/optuna_search.py

```diff
@@ -362,6 +370,17 @@ def __init__(
         self._sampler = sampler
         self._seed = seed
 
+        if storage:
+            assert (
+                study_name
```

Comment:
Why do we have to specify a `study_name` to use the optuna storage? Can we use the default study name `optuna` as being used in the `InMemoryStorage` by default? What if I passed `study_name=optuna` to initialize the OptunaSearcher?